plugins/out_stackdriver/stackdriver_conf.c

```diff
@@ -420,9 +420,16 @@ struct flb_stackdriver *flb_stackdriver_conf_create(struct flb_output_instance *
     else {
         if (ctx->is_k8s_resource_type == FLB_TRUE) {
             ctx->tag_prefix = flb_sds_create(ctx->resource);
+            ctx->tag_prefix = flb_sds_cat(ctx->tag_prefix, ".", 1);
         }
     }
 
+    /* Custom Regex */
+    tmp = flb_output_get_property("custom_regex", ins);
```

Comment:
Looks like we would expose a new configuration of the stackdriver output plugin. `custom_regex` might be a bit generic. What about something like `custom_tag_regex` if it's specific to tags?

Reply:
BTW do we need to update https://docs.fluentbit.io/manual/pipeline/outputs/stackdriver accordingly?

Reply:
Good call. `custom_tag_regex ` sounds better. And yes I will update the doc once this PR is merged.

Reply:
Done.